test.js

```diff
@@ -185,3 +185,56 @@ test('ensure different promises are returned on second call with the same select
 	document.querySelector('.unicorn').remove();
 	t.is(prependElement(), await elementReady('.unicorn'));
 });
+
+test('subscribe: check if elements are detected', async t => {
```

Comment:
There needs to be tests for the `timeout` and `stopOnDomReady` options with `subscribe` too.